flink-end-to-end-tests/test-scripts/elasticsearch-common.sh

```diff
@@ -87,6 +87,8 @@ function verify_result_hash {
   local numRecords=$3
   local hash=$4
 
+  sleep 20s
```

Comment:
@twalthr After many local and Travis tests, I determined that "Executing SQL: Values -> Elasticsearch (upsert)" is very unstable (statistics count results). I think it's because of the prematurely getting results from Elasticsearch. Here I feel that waiting for a while before getting results from Elasticsearch will reduce the chances of failure. cc @pnowojski

Reply:
OK, I will create a new issue.

Reply:
Ops, sorry. I misunderstood @twalthr comment. I agree with him that we shouldn't be merging this `sleep 20s` fix, especially that the proper fix should easy (test for the results in a loop). I will look into it.